apps/site-nft-landing/firebase.json

```diff
@@ -9,12 +9,12 @@
     "rewrites": [
       {
         "source": "**",
-        "destination": "/index.html"
+        "destination": "/en.html"
       }
     ],
     "headers": [
       {
-        "source": "/index.html",
```

Comment:
seemed nicer to explicitly name the files - shows up in the browser as /en - /fr, which seems nicer.